include/mgba/internal/gba/hardware-extensions.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2021 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#ifndef HARDWARE_EXTENSIONS_H
+#define HARDWARE_EXTENSIONS_H
+
+#include <mgba-util/common.h>
+
+#include <mgba/core/log.h>
+#include <mgba/core/timing.h>
+
+#include <mgba/internal/gba/io.h>
+#include <mgba/internal/gba/hardware-extensions-ids.h>
+
+
```

Comment:
Extraneous line.

Reply:
I don't understand what you mean by extraneous... As in "strange" line? Or did you just mean to say there's an extra blank line?

Reply:
There's an extra blank line, yes.

Reply:
Fixed.